mitiq/ddd/insertion.py

```diff
@@ -83,3 +85,40 @@ def get_slack_matrix_from_circuit_mask(mask: np.ndarray) -> np.ndarray:
                         break
 
     return slack_matrix
+
+
+@noise_scaling_converter
+def insert_ddd_sequences(
+    circuit: Circuit,
```

Comment:
```suggestion
    circuit: QPROGRAM,
```
I would suggest this change. I know that in other places we use `Circuit`, but `QPROGRAM` is probably more appropriate.

Reply:
Ah, fair point. I added a second function that takes `QPROGRAM` (with docstring) and is external, with the existing function made internal (but still using the noise scaling converter). How's that?

Reply:
Sounds good!